_pages/dev/performance/deferred-loading.md

```diff
@@ -8,24 +8,24 @@ title: Deferred Loading (DRAFT)
 
 {% include docs/feature_version.html content=version_note %}
 
-Deferred Loading is a technique to postpone rendering of CMS Components. Components which are outside the user's visible area (the *viewport*), will not be rendered in advance. This technique is introduced to improve the initial rendering of the complete page. This does not only saves the creating of components but also defers further loading of component specific data.
+Deferred loading is a technique that is used to postpone the rendering of CMS components. Any component that is outside the viewport (that is, the area that is visible to the user) is not rendered in advance. This improves the initial rendering of the complete page. Deferred loading not only defers the creation of components, it also defers the loading of component-specific data.
```

Comment:
"it also defers the loading of component-specific data."
can we change that to:

"it also defers the loading of of any component specific data such as media or back end calls ."

Reply:
Done.